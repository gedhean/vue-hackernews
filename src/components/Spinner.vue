<template>
  <transition>
    <svg class="spinner" width="50px" height="50px" viewBox="0 0 50 50">
      <circle
        class="path"
        fill="none"
        stroke-width="4"
        stroke-
        linecap="round"
        cx="22"
        cy="22"
        r="20"
      ></circle>
    </svg>
  </transition>
</template>
