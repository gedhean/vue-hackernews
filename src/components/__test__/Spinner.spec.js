import { shallowMount } from "@vue/test-utils";
import Spinner from "../../components/Spinner.vue";

describe("Spinner.vue", () => {
  test("renders correctly", () => {
    const wrapper = shallowMount(Spinner);

    expect(wrapper).toMatchSnapshot();
  });
});
