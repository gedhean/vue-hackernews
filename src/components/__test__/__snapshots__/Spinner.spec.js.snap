// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner.vue renders correctly 1`] = `
<transition-stub><svg width="50px" height="50px" viewBox="0 0 50 50" class="spinner">
    <circle fill="none" stroke-width="4" stroke-="" linecap="round" cx="22" cy="22" r="20" class="path"></circle>
  </svg></transition-stub>
`;
